plugins/git/commands

```diff
@@ -21,7 +22,8 @@ git_build_app_repo() {
   chmod 755 "$GIT_BUILD_APP_REPO_TMP_WORK_DIR"
   unset GIT_DIR GIT_WORK_TREE
   pushd "$GIT_BUILD_APP_REPO_TMP_WORK_DIR" > /dev/null
-  [[ ! -d "$DOKKU_ROOT/$APP" ]] && apps_create "$APP"
+  [[ ! -d "$DOKKU_ROOT/$APP" ]] && DOKKU_FIRST_DEPLOY=true
```

Comment:
Yeah I thought about that. Not sure what the best way to deal with that is. Potentially create a sentinel file (or use the new config stuff) to track whether its new or not.